_docs/setup/kubernetes/faq.md

```diff
@@ -9,18 +9,25 @@ type: markdown
 ---
 {% include home.html %}
 
-* _How do I check if my cluster has enabled the alpha feature required for automatic sidecar injection?_
+* _How do I check if my cluster has enabled the alpha features required for automatic sidecar injection?_
 
-  Automatic sidecar injection requires the [initilizer alpha feature](https://kubernetes.io/docs/admin/extensible-admission-controllers/#enable-initializers-alpha-feature).
-  Run the following command to check if the initializer has been enabled (empty output indicates that initializers are not enabled):
+  Automatic sidecar injection requires the
+  [initilizer alpha feature](https://kubernetes.io/docs/admin/extensible-admission-controllers/#enable-initializers-alpha-feature).
+  Run the following command to check if the initializer has been enabled
+  (empty output indicates that initializers are not enabled):
  
   ```bash
   kubectl api-versions | grep admissionregistration
   ```
```

Comment:
FYI, this is not sufficient to determine whether initializers are available in the cluster. The `Initializer` plugin must also be enabled in flags to apiserver. I don't know of a way to discover this flag via `kubectl`.